codejam-canvas/script.js

```diff
@@ -0,0 +1,47 @@
+const canvas = document.querySelector('#canvas');
+const ctx = canvas.getContext('2d');
+let inputs = document.querySelectorAll('input[name="size"]');
+for (let i = 0; i < inputs.length; i++) {
+    inputs[i].addEventListener('change', function () {
```

Comment:
Here on each event listener binding you create new function, you can declare this function once and apply here each time this one instance of function